dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.h

```diff
@@ -589,7 +592,7 @@ class OpenDDS_Rtps_Udp_Export RtpsUdpDataLink : public DataLink, public virtual
 
   struct CountKeeper {
     OPENDDS_MAP(size_t, IdCountSet) heartbeat_counts_;
-    IdCountSet acknack_counts_;
+    OPENDDS_MAP_CMP(EntityId_t, IdCountSet, EntityId_tKeyLessThan) acknack_counts_;
```

Comment:
Make a typedef above for readability / in case we need to use iterators in the future?